connector/loki/loki.go

```diff
@@ -82,19 +83,26 @@ func (client *LokiConnector) IsReady() bool {
 	return err == nil && response.StatusCode == 200
 }
 
-func CreateLokiConnector(logger *logging.Logger, address string, maxWaitTime time.Duration, batchSize int64) (*LokiConnector, error) {
+//CreateLokiConnector creates a loki connector
+func CreateLokiConnector(logger *logging.Logger,
+	address string,
+	maxWaitTime time.Duration,
+	batchSize int64,
+	tenantID string) (*LokiConnector, error) {
+
 	client := LokiConnector{
 		url:         address,
 		maxBatch:    batchSize,
 		maxWaitTime: maxWaitTime,
 		quit:        make(chan struct{}),
 		streams:     make(chan *LokiStream),
 		logger:      logger,
-		endpoints:   endpoints{
```

Comment:
Please don't break indentation. Linter will complain.

Reply:
I'm not sure why, but if I keep the indentation gofmt -s complains (that's why I made the change on this line). This is the result of `gofmt -s -d` with your suggested change:
```
--- connector/loki/loki.go.orig	2021-07-12 16:43:53.246017361 +0200
+++ connector/loki/loki.go	2021-07-12 16:43:53.246017361 +0200
@@ -98,7 +98,7 @@
 		streams:     make(chan *LokiStream),
 		logger:      logger,
 		tenantID:    tenantID,
-		endpoints:   endpoints{
+		endpoints: endpoints{
 			push:  "/loki/api/v1/push",
 			query: "/loki/api/v1/query_range",
 			ready: "/ready",
```

Reply:
I think the linter we use with our CI just uses `gofmt -s`, right?